pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -72,50 +69,28 @@ func OpenInMemory() (db *DB, err error) {
 	db = &DB{
 		DB: sqlite,
 	}
-	if err := db.init(); err != nil {
-		return nil, utils.CombineErrors(err, db.DB.Close())
-	}
 
 	return db, nil
 }
 
-func (db *DB) init() (err error) {
-	tx, err := db.DB.Begin()
-	if err != nil {
-		return err
-	}
-
-	defer func() { _ = tx.Rollback() }()
-
-	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `ttl` (`id` BLOB UNIQUE, `created` INT(10), `expires` INT(10), `size` INT(10));")
-	if err != nil {
-		return err
-	}
-
-	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bandwidth_agreements` (`satellite` BLOB, `agreement` BLOB, `signature` BLOB);")
-	if err != nil {
-		return err
+// Migration define piecestore DB migration
+func (db *DB) Migration() *migrate.Migration {
+	migration := &migrate.Migration{
+		Table: "versions",
+		Steps: []*migrate.Step{
+			{
+				Description: "Initial setup",
+				Version:     0,
+				Action: migrate.SQL{
```

Comment:
If there are more than two actions are there what is the preferred way to implement?  Should we do it as separate steps or two actions in action func()?

Reply:
General rule can be thought of as 1 version per commit... of course sometimes it might be easier to separate them into different versions.